rskj-core/src/main/java/co/rsk/mine/MinerServerImpl.java

```diff
@@ -504,7 +506,12 @@ private Block createBlock(Block newBlockParent, List<BlockHeader> uncles, List<T
         @Override
         public void run() {
             Block bestBlock = blockchain.getBestBlock();
-            buildBlockToMine(bestBlock, false);
+            try {
+                buildBlockToMine(bestBlock, false);
+            } catch (Throwable th) {
+                logger.error("Unexpected error: ", th);
+                panicProcessor.panic("mserror", th.toString());
```

Comment:
Please use `th.getMessage()` for consistency.

Reply:
done!